ipyparallel/meta.yaml

```diff
@@ -0,0 +1,84 @@
+package:
+  name: ipyparallel
+  version: "4.1.0"
+
+source:
+  fn: ipyparallel-4.1.0.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipyparallel/ipyparallel-4.1.0.tar.gz
+  md5: 56acdbcef05d3a6adbc987abc38174db
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - ipyparallel = ipyparallel:main
+    #
+    # Would create an entry point called ipyparallel that calls ipyparallel.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - ipython_genutils
+    - decorator
+    - pyzmq >=13
+    - ipython >=4
+    - jupyter_client
+    - ipykernel
+    - nose
+    - notebook
+    - tornado >=4
```

Comment:
Please remove the last three dependencies, i.e. nose, notebook and tornado. They are not needed to build or run this package.